files/build_templates/docker_image_ctl.j2

```diff
@@ -197,6 +197,17 @@ start() {
 {%- endif %}
 {%- if sonic_asic_platform == "mellanox" %}
     # TODO: Mellanox will remove the --tmpfs exception after SDK socket path changed in new SDK version
+{%- endif %}
+{%- if docker_container_name == "snmp" %}
+    NET_NS_MOUNT=""
+    if [ -d "/var/run/netns/" ]; then
+	NS_PATH="/var/run/netns/*"
```

Comment:
>/var/run/netns/* [](start = 10, length = 16)

How about mounting the parent directly?

Reply:
Each namespace inside /var/run/netns is a symlink to /proc/<PID>/ns/net of database docker process. If we mount the /var/run/netns directory, then we will have to mount  /proc/<PID>/ns/net for each namespace which cannot be done easily as mount /proc/ file system from host to docker is not allowed. Hence, while mounting, "/proc/<PID>/ns/net" is mounted as /var/run/netns/<namespace> inside docker for each namespace.